src/CLIPreTasks/OutdatedChecker/OutdatedChecker.ts

```diff
@@ -23,11 +24,18 @@ export class OutdatedChecker {
       return
     }
 
-    const errMsg = chalk.bold(
-      `This version ${pkgJson.version} is outdated. Please update to the latest version: ${chalk.green(
-        'yarn global add vtex'
+    const errMsg =
+      `${chalk.bold(
+        `Your Toolbelt version (${pkgJson.version}) is outdated`
+      )}. You must update using the same method you used to install. Here are some examples:` +
+      `\n\n` +
+      `• If you installed using ${ColorifyConstants.COMMAND_OR_VTEX_REF(
+        `yarn`
+      )}, update running ${ColorifyConstants.COMMAND_OR_VTEX_REF(`yarn global add vtex`)}.` +
+      `\n\n` +
+      `• If you installed using our new method there is in alpha-version, update running ${ColorifyConstants.COMMAND_OR_VTEX_REF(
+        `vtex autoupdate`
```

Comment:
This `errMsg` is duplicates. Can you define it in only one place, so it's easier to change it if needed?

Reply:
Actually is not the same, only differ by `deprecated` or `outdated`

Reply:
Maybe creating a base message and receive an argument of the status is a better solution, what do you think about it?

Reply:
Think its good! If there is a new way of installing it, for instance, it will make the refactor of this text much easier